Sources/ClientRuntime/Retries/DefaultRetryStrategy/DefaultRetryToken.swift

```diff
@@ -11,15 +11,12 @@ import struct Foundation.TimeInterval
 ///
 /// The retry token contains all the state relevant to one request that is needed to manage retry
 /// until the request succeeds or fails after zero or more retries.
-public class DefaultRetryToken: RetryToken {
+public final class DefaultRetryToken: RetryToken {
 
     /// The number of retry attempts that have been made using this token.
     /// Defaults to zero at the initial attempt, goes up by one for each subsequent attempt.
     public internal(set) var retryCount: Int = 0
 
-    /// The delay, in seconds, to the next retry.
-    public internal(set) var delay: TimeInterval?
```

Comment:
This delay param on token is not used in the Swift implementation, hence it is removed.